fastlane/Fastfile

```diff
@@ -350,10 +354,10 @@ platform :android do
         "android.injected.signing.key.alias" => ENV["KEY_ALIAS"],
         "android.injected.signing.key.password" => ENV["KEY_PASS"],
       })
-    sh("cp", "../android/app/build/outputs/bundle/#{build_type}/app-#{build_type}.aab", "../")
-    sh("mkdir", "-p", "../bundle")
-    sh("cp", "../android/app/build/generated/assets/createBundle#{build_type.capitalize}JsAndAssets/index.android.bundle", "../bundle")
-    sh("cp", "../android/app/build/intermediates/sourcemaps/react/#{build_type}/index.android.bundle.packager.map", "../bundle/index.android.bundle.map")
+      sh("cp", "../android/app/build/outputs/bundle/#{app_flavor}/#{build_type}/app-#{app_flavor}-#{build_type}.aab", "../#{ENV["AAB_FILE_NAME"]}")
```

Comment:
Hey @jorelosorio, do you know if this line is correct? It seems to crash when building the FRAM app as we speak, seemingly because `app-` is included twice in the file name: 

https://github.com/AtB-AS/mittatb-app/actions/runs/6876585266/job/18704550174#step:17:2011

Reply:
Added a PR that could possibly help here: https://github.com/AtB-AS/mittatb-app/pull/4042